integration_test/third_party_apps_test/applications/dcgm/centos_rhel/install

```diff
@@ -18,24 +18,32 @@ install_cuda_from_runfile() {
     # Remove existing installation before using the runfile
     remove_cuda_package
     remove_driver_package
+    # For Rocky Linux 9: when a new OS version becomes available, the default 
+    # repo setting (/etc/yum.repos.d/rocky.repo) will automatically point to the
+    # new version's repo. This is problematic since the new OS is not available
+    # right away on GCE. Set up the matched repo to install the correct 
+    # kernel-devel-$(uname -r)
+    # Not needed for RL8 since 8.10 is already the last RL8 release.  
+    if [[ $ID == rocky && "${MAJOR_VERSION_ID}" == 9 ]]; then
+        cat <<EOF | sudo tee /etc/yum.repos.d/rocky-matched.repo
+[appstream-matched]
+name=Rocky Linux \$releasever - AppStream - Matched
+baseurl=https://dl.rockylinux.org/vault/rocky/$VERSION_ID/AppStream/\$basearch/os/
+gpgcheck=1
+enabled=1
+countme=1
+metadata_expire=6h
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-Rocky-9
+EOF
+    fi
     sudo yum install -y kernel-devel-$(uname -r) pciutils gcc make wget yum-utils
 
     # Installing latest version of NVIDIA CUDA and driver
-    # Data Center/Tesla drivers and CUDA are released on different schedules;
-    # normally we install the matching versions of driver and CUDA
-    # ($DRIVER_VERSION == $CUDA_BUNDLED_DRIVER_VERSION); due to https://github.com/NVIDIA/open-gpu-kernel-modules/issues/550
```

Comment:
The mentioned issue has been fixed in the new version of CUDA, so here we can simplify the steps to install CUDA+driver from one single package.